tests/foreman/cli/test_contentview.py

```diff
@@ -4917,27 +4751,27 @@ def test_positive_arbitrary_file_repo_addition(self):
 
         :expectedresults: Check FR is added to CV
 
-        :CaseAutomation: NotAutomated
+        :CaseAutomation: notautomated
```

Comment:
Double check your testimony tokens here. The case is now enforced, and is causing failures in the GH workflow.

Second - this one is wrong! Not sure when these were automated and this was originally missed, but now's as good a time as ever to correct them.

I would recommend setting `:CaseAutomation: Automated` at the module level, and setting `NotAutomated` specifically on stubbed cases.